test/functional/feature_taproot.py

```diff
@@ -1240,8 +1240,8 @@ def sample_spenders():
     # Create a list of scripts which will be built into a taptree
     scripts = [
         # leaf label, followed by CScript
-        ("encodeable_pushdata1", CScript([OP_DROP, OP_PUSHDATA1, b'aa' * 75])),
-        ("nonstd_encodeable_pushdata1", CScript([OP_PUSHDATA1, b'aa'])),
+        ("2byte_pushdata", CScript([OP_DROP, b'\xaa\xaa'])),
+        ("nonstd_2byte_pushdata", CScript.fromhex("4c02aaaa")),
```

Comment:
I'm guessing it's intentional to have two different ways of creating `CScript` here? 
For better illustration.

Reply:
The python framework won't let you construct a non-standard serialization natively, AFAIK. So we're doing it manually.